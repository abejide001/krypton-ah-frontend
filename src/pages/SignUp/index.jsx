import React from 'react';
import ModalComponent from '../../components/FormContainer';
import SignUpForm from '../../components/SignUpForm';

function SignUp() {
  return (
    <ModalComponent
      title="sign up"
      baseText="Already have an account?"
      link="Sign In"
      formLink="signin">
      <SignUpForm />
    </ModalComponent>
  );
}

export default SignUp;
