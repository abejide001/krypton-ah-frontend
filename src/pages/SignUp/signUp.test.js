import React from 'react';
import { shallow } from 'enzyme';
import SignUp from '.';


describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<SignUp />);
    expect(wrapper).toMatchSnapshot();
  });
});
