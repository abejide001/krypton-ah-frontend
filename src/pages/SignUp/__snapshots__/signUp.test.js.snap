// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<ModalComponent
  baseText="Already have an account?"
  formLink="signin"
  link="Sign In"
  title="sign up"
>
  <SignUpForm />
</ModalComponent>
`;
